src/content/translations/es/what-is-ethereum/index.md

```diff
@@ -3,7 +3,7 @@ title: Principiantes
 lang: es
 ---
 
-# :wave: ¡Bienvenidos! {#-welcome}
```

Comment:
There's a bunch of other translations with the `:wave:` emoji that should be replaced. Search the codebase for that.